migrations/sql/V2019.03.25.11.06__add_hsrc_code.sql

```diff
@@ -0,0 +1,1069 @@
+CREATE TABLE IF NOT EXISTS article_act_code
+(
+    article_act_code character varying(5)     PRIMARY KEY                        ,
+    description      character varying(100)                              NOT NULL,
+    effective_date   date                     DEFAULT now()              NOT NULL,
+    expiry_date      date                     DEFAULT '9999-12-31'::date NOT NULL,
+    update_timestamp timestamp with time zone DEFAULT now()              NOT NULL
+);
+
+
+-- Comments
+COMMENT ON TABLE article_act_code IS 'Contains a list of values used to indicate an Act or Regulation and is used to define the article_act_code. I.e. MA - Mines Act, HSRC - Health, Safety and Reclamation Code for Mines in BC.';
```

Comment:
I think I made a mistake when providing an example of a comment for the article_act_code table. The last bit doesn't make as much sense today... Can you please update it?

"Contains a list of values used to indicate an Act or Regulation. I.e. MA - Mines Act, HSRCM - Health, Safety and Reclamation Code for Mines in BC."